pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -210,32 +211,58 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 
 		sort.Stable(osgraph.ByKey(allMarkers))
 		sort.Stable(osgraph.ByNodeID(allMarkers))
+
+		errors := 0
 		if errorMarkers := allMarkers.BySeverity(osgraph.ErrorSeverity); len(errorMarkers) > 0 {
 			fmt.Fprintln(out, "Errors:")
 			for _, marker := range errorMarkers {
+				errors++
 				fmt.Fprintln(out, indent+"* "+marker.Message)
+				if len(marker.Suggestion) > 0 {
+					if d.Suggest {
+						fmt.Fprintln(out, indent+"  "+marker.Suggestion.String())
+					}
+				}
 			}
 		}
+
+		warnings := 0
 		if warningMarkers := allMarkers.BySeverity(osgraph.WarningSeverity); len(warningMarkers) > 0 {
-			fmt.Fprintln(out, "Warnings:")
-			for _, marker := range warningMarkers {
-				fmt.Fprintln(out, indent+"* "+marker.Message)
+			if d.Suggest {
+				fmt.Fprintln(out, "Warnings:")
 			}
-		}
-		if infoMarkers := allMarkers.BySeverity(osgraph.InfoSeverity); len(infoMarkers) > 0 {
-			fmt.Fprintln(out, "Info:")
-			for _, marker := range infoMarkers {
-				fmt.Fprintln(out, indent+"* "+marker.Message)
+			for _, marker := range warningMarkers {
+				warnings++
+				if d.Suggest {
+					fmt.Fprintln(out, indent+"* "+marker.Message)
+					if len(marker.Suggestion) > 0 {
+						fmt.Fprintln(out, indent+"  "+marker.Suggestion.String())
+					}
+				}
 			}
 		}
 
-		fmt.Fprintln(out)
+		if errors != 0 || (d.Suggest && warnings != 0) {
+			fmt.Fprintln(out)
+		}
 
-		if (len(services) == 0) && (len(standaloneDCs) == 0) && (len(standaloneImages) == 0) {
+		switch {
+		case !d.Suggest && errors > 0:
```

Comment:
Sorry, I think this has to get uglier.  If I ran `oc status` and got 1 error and 3 warnings, I'm going to want to know about the number of warnings too.